com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/resources/jsweb/dmloader.js

```diff
@@ -992,16 +984,16 @@ var Module = {
     }],
 
     _preloadAndCallMain: function() {
-        // If the archive isn't loaded,
-        // we will have to wait with calling main.
-        if (!Module._archiveLoaded) {
-            Module._waitingForArchive = true;
-        } else {
-            Module.preloadAll();
-            if (Module._isEngineLoaded) {
-                // "Starting...."
-                ProgressUpdater.complete();
-                Module._callMain();
+        if (Module._syncInitial || Module.persistentStorage != true) {
```

Comment:
Also add check if we don't have persistent storage. It means that `_syncInitial` never become `true`.